filter/eq.go

```diff
@@ -0,0 +1,93 @@
+package filter
+
+import "time"
+
+// EqInt64 composes 'equal' operation from int64 value.
+// Result is equivalent to: field == value
+func EqInt64(field string, value int64) expr {
+	return expr{field: comparison{Eq: value}}
+}
+
+// EqInt composes 'equal' operation from Int value.
+// Result is equivalent to: field == value
+func EqInt(field string, value int) expr {
+	return expr{field: comparison{Eq: value}}
+}
+
+// EqUint composes 'equal' operation from uint value.
+// Result is equivalent to: field == value
+func EqUint(field string, value uint) expr {
```

Comment:
do we need to support `uint`? Let's stick to whatever types we support.

Reply:
It's automatically converted to our int32 and back, no need to limit users.
There is no support for Uint64 for example because we don't have enough precision to store it.